mava/components/jax/training/model_updating.py

```diff
@@ -130,8 +129,7 @@ def model_update_minibatch(
                     gradients[agent_key]
                 )
                 agent_metrics[agent_key]["norm_updates"] = optax.global_norm(updates)
-                metrics[agent_key] = agent_metrics
-            return (params, opt_states), metrics
+            return (params, opt_states), agent_metrics
```

Comment:
@arnupretorius @KaleabTessera this seems like a MASSIVE bug which has how been fixed. No idea how our systems were even training with this.